### ISSUE TITLE
ONLY pop up the notification of a new version IF it is compatible with the currently running version of Mac OS.

### ISSUE THREAD

**Issue by smayer97** [crystalidea/macs-fan-control#423]
One change I would ask is that you only pop up a notification of new versions on supported/compatible versions of the Mac OS. I currently get pop ups of new versions of Macs Fan Control one of my Macs that is running OS X.6.8 and clearly the new versions are not compatible, so being able to stop receiving those notices would be appreciated. (current version on that computer is v1.2.1).

I know that I could opt to not receive any notifications but if I upgrade mac OS then I would have to remember to turn this back on. So if this was automated, that would be better.

So in other words, only pop up the notification of a new version IF it is compatible with the currently running version of Mac OS.

**Comment by kleuter** [comment]
This issue has been automatically closed since there has not been any recent activity. Please re-open it if needed.
